raiden/network/transport/matrix/transport.py

```diff
@@ -273,7 +278,7 @@ class MatrixTransport(Runnable):
     def __init__(self, config: dict):
         super().__init__()
         self._config = config
-        self._raiden_service: RaidenService = None
+        self._raiden_service: Optional[RaidenService] = None
```

Comment:
IIRC we said that we won't use `Optional` for arguments that have a default of `None` (since that already expresses the optional-ness)

Reply:
Correct, @LefterisJP posted about it on RC if i remember correctly.

Reply:
From what I could see on @konradkonrad 's debugging, seems like mypy doesn't detect the `Optional` part for variable type declaration, only for parameters, that's why it's required to explicitly state it there

Reply:
Then we should probably change our code style and always use Optional.

Reply:
Hmm our code style is Optional being "optional" only in function args.

Reply:
Check https://github.com/raiden-network/raiden/blob/master/CONTRIBUTING.md for this